tasks/build-scan-push/build-scan-push.yaml

```diff
@@ -50,14 +50,21 @@ spec:
       else
       echo "No vulnerabilities found."
       fi
-  - image: ghcr.io/alexkondrat/container-tools:latest
-    name: push-sign-verify
+  - image: gcr.io/go-containerregistry/crane:debug
```

Comment:
You can use the tagged version for debug: https://console.cloud.google.com/artifacts/docker/go-containerregistry/us/gcr.io/crane%2Fdebug/sha256:66336f608f9a219c93fdb610fbcb491b5d38e72c250a191f2c5efda6427028c1